keyboards/stardustn0/keyboard.json

```
@@ -0,0 +1,175 @@
+{
+    "manufacturer": "Stars",
+    "keyboard_name": "StarDustN0",
+    "maintainer": "navidyt",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP8","GP11","GP19","GP18","GP17","GP16"],
+        "rows": ["GP3","GP9","GP13","GP14","GP15"]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0x4E54"
+    },
+    "split": {
+        "enabled": true,
+        "main": "left",
+        "soft_serial_pin": "GP1",
+        "transport": {
+            "protocol": "serial"
+        },
+        "matrix_pins": {
+            "right": {
+                "cols": ["GP15","GP14","GP13","GP12","GP20","GP26"],
+                "rows": ["GP28","GP22","GP18","GP17","GP16"]
+            }
+        },
+        "usb_detect": {
+            "timeout": 2500
+        }
+    },
+    "layouts": {
+        "LAYOUT_split_2x28": {
```

`LAYOUT` would likely be fine here. 
Also given existing naming convention, this should be:
```suggestion
        "LAYOUT_split_4x6_4": {
```